.github/workflows/main.yml

```diff
@@ -9,44 +9,60 @@ jobs:
     # ================================================
 
     eslint:
-        name: "ESLint (Node ${{ matrix.node }})"
+        name: "ESLint (Node ${{ matrix.node == '*' && 'latest' || matrix.node }})"
```

Comment:
"*" is not the same as latest, as I understand it. (If it was, we wouldn't need to change this.) IMHO `*` here would be better here: whilst not terribly clear for those who don't know what it means, it is at least not actively misleading.